src/components/WalletSelector/multichain/tezos/TezosAddWallet.tsx

```diff
@@ -43,7 +44,12 @@ export const TezosAddWallet = ({ queryRef, reset }: Props) => {
   const [error, setError] = useState<Error>();
   const [account, setAccount] = useState<string>();
   const [publicKey, setPublicKey] = useState<string>();
-  const beaconClient = useBeaconState();
+  const [wallet, setWallet] = useState<string>();
+  const { requestPermissions, requestSignature } = useBeaconActions();
+
+  const messageHeaderText = useMemo(() => {
```

Comment:
nit: no need for useMemo for primitive types

Reply:
Updated!